manim/mobject/number_line.py

```diff
@@ -100,6 +98,7 @@ def __init__(
         numbers_to_include=None,
         # temp, because DecimalNumber() needs to be updated
         number_scale_value=0.75,
+        include_tick_at_origin=True,
```

Comment:
This is probably quite nitpicky, but since this defaults to `True`, I feel like it would make more sense to be written as ```exclude_origin_tick = False```. Also, for the sake of organization, it should be under the `#ticks` section of the parameters.

Reply:
Sorry I didn't notice there was a ticks section, moved there now and made those other changes